### PR TITLE
EN: UI Capitalization, Battle Arena

### PR DESCRIPTION
**Pull request by ordy**
- Capitalized And Fixed UI Titles And Entries
- Translated some BA content

### CODE REVIEW

File: UI/ui_myroom.csv [hunk 1]
```diff
@@ -131,53 +131,53 @@ RoomSize_Delete#0,"""Delete"""
...
-RoomGoods_Custmize_Remove#0,"""Undo combination"""
-RoomGoods_MessageInOut#0,"""Edit message"""
+RoomGoods_Custmize_Gimmick#0,"""Combine Goods"""
+RoomGoods_Custmize_Remove#0,"""Undo Combination"""
+RoomGoods_MessageInOut#0,"""Edit Message"""
 RoomGoods_PresentBox#0,"""Open the $(0)"""
 RoomGoods_GoodJob#0,"""Send a Good Job"""
-RoomGoods_GoodJob_Greeting#0,"""Edit greeting"""
-RoomGoods_GoodJob_Thanks#0,"""Edit thanks message"""
+RoomGoods_GoodJob_Greeting#0,"""Edit Greting"""
```
Comment (alama) [inline UI/ui_myroom.csv]: Greting? not Greeting?

File: UI/ui_myroom.csv [hunk 2]
```diff
@@ -363,21 +363,21 @@ ob_1052_0000#0,"""予備枠"""
...
 Title#0,"""Configure Admission Settings"""
 If_All_OK#0,"""All players"""
-If_Friend_Guild_Member#0,"""Friends & Team Members only"""
-If_Guild_Member#0,"""Team Members only"""
-If_Friend_Member#0,"""Friends only"""
-If_All_Not#0,"""Private (only you may enter)"""
+If_Friend_Guild_Member#0,"""Friends & Team Members Only"""
+If_Guild_Member#0,"""Team Members Only"""
+If_Friend_Member#0,"""Friends Only"""
+If_All_Not#0,"""Private (only uou may enter)"""
```
Comment (alama) [inline UI/ui_myroom.csv]: You, not uou?

**Comment by ordy** [comment]
Already fixed that and some other things on local, need to translate/fix a few more lines then I'll update the PR.

_Sent from my Xiaomi Redmi Note 3 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Review by SynthSy — APPROVED** [review]

**Comment by ordy** [comment]
Done, review and ready to merge.

**Pull request merged**